core/meta/src/TClass.cxx

```diff
@@ -3451,38 +3451,31 @@ const char *TClass::GetSharedLibs()
 
 TList *TClass::GetListOfBases()
 {
-   if (!fBase) {
-      if (fCanLoadClassInfo) {
-         if (fState == kHasTClassInit) {
+   if (fBase)
```

Comment:
Yes, it's a lot faster. I edited this function because it appeared as a bottleneck in VTune.